src/main/java/com/epam/lab/paymentsystem/controller/CardController.java

```diff
@@ -32,4 +45,40 @@ public String getAccountPage(@PathVariable(name = "accountId") long id, Model mo
     model.addAttribute("cardList", cards);
     return ACCOUNT_PAGE;
   }
+
+  /**
+   * Returns add new card page.
+   *
+   * @param model model
+   * @return card add page view
+   */
+  @GetMapping(value = "/user/account/{accountId}/addCard")
+  public String getAddCardPage(Model model) {
+    LOGGER.info("Access to card creation page");
+    model.addAttribute("userList", userService.getAllUsers());
+    return ADD_CARD_PAGE;
+  }
+
+  /**
+   * Add card form and send it to service layer.
+   *
+   * @param accountId account
+   * @param cardLabel label
+   * @param userLogin login
+   * @param model model
+   * @return JSP view
+   */
+  @PostMapping(value = "/user/account/{accountId}/addCard")
+  public String addCard(
+      @PathVariable(name = "accountId") long accountId,
+      @RequestParam(name = "label") String cardLabel,
+      @RequestParam(name = "login") String userLogin,
+      Model model) {
+
+    CardDto cardDto = new CardDto(0, accountId, userLogin, cardLabel, false);
```

Comment:
you can create cardDto with help of thymeleaf and take it as argument in controller

Reply:
Done:
1) Added dto (new CardDto()) attribute to model in getAddCardPage() method.
2) Replace 2 params with 1 dto via @ModelAttribute annotation.